contrib/debian/copyright

```diff
@@ -2,12 +2,12 @@ Format: http://svn.debian.org/wsvn/dep/web/deps/dep5.mdwn?rev=174
 Upstream-Name: Bitcoin
 Upstream-Contact: Satoshi Nakamoto <satoshin@gmx.com>
  irc://#bitcoin@freenode.net
-Source: https://github.com/bitcoin/bitcoin
+Source: https://github.com/bitcoinclassic/bitcoinclassic
 
 Files: *
-Copyright: 2009-2012, Bitcoin Core Developers
+Copyright: 2009-2012, Bitcoin Classic Developers
```

Comment:
On what is copyright being claimed here and can the copyright belong to both Core and Classic developers?